File: app_sample_java/src/main/java/com/joelzhu/lib/scanner/java/commands/CommandRun.java
package com.joelzhu.lib.scanner.java.commands;

import com.joelzhu.lib.scanner.annotation.CompileScan;
import com.joelzhu.lib.scanner.java.ICommand;

/**
 * Command: human to run.
 *
 * @author JoelZhu
 * @since 2023-01-27
 */
@CompileScan(priority = 4)
public final class CommandRun implements ICommand {
    @Override
    public String execute() {
        return "Running...";
    }
}

File: app_sample_multimodule_lib1/src/main/java/com/joelzhu/lib/scanner/multimodule/commands/CommandRun.kt
package com.joelzhu.lib.scanner.multimodule.commands

import com.joelzhu.lib.scanner.annotation.CompileScan
import com.joelzhu.lib.scanner.multimodule.ICommand

/**
 * Command: human to run.
 *
 * @author JoelZhu
 * @since 2023-03-13
 */
@CompileScan(priority = 4)
class CommandRun : ICommand {
    override fun execute(): String {
        return "Running... From module1, priority = 4"
    }
}